src/emulator/gui/src/ui_main_menubar.cpp

```diff
@@ -38,6 +38,21 @@ void DrawMainMenuBar(HostState &host) {
             ImGui::EndMenu();
         }
 
+        if (ImGui::BeginMenu("Configuration")) {
+            ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+            if (ImGui::BeginMenu("Emulation Mode")) {
+                if (ImGui::MenuItem("PSVita", nullptr, !host.cfg.pstv_mode)) {
+                    host.cfg.pstv_mode = false;
+                }
+                if (ImGui::MenuItem("PSTV", nullptr, host.cfg.pstv_mode)) {
```

Comment:
`PS TV`

Reply:
i think i've never seen PSTV named as PS TV. Either PSTV or Playstation TV.

Reply:
![image](https://user-images.githubusercontent.com/26081953/50773059-5f9d8b00-1298-11e9-9e6b-170b8ef18122.png)
I have.

Reply:
Yea @Rinnegatamante it's the official name for it, I looked it up.
Well, PlayStation TV is, at least. Maybe we should use full names?